connector/mqtt/locust/src/ejbca/certificate.py

```diff
@@ -95,15 +59,11 @@ def sign_cert(self) -> str:
         """
```

Comment:
This method functionality does not seem to reflect its name and description. As I can see, it could be named `generate_certificate`.